src/terraform.yml

```
@@ -18,10 +18,11 @@
           - ansible
           - boto3
   tasks:
-    - name: Install the AWS CLI tool
+    - name: Install git and the AWS CLI tool
```

Total nitpick here, but I'm sure you can appreciate that it looks odd to write "Install git and the AWS CLI tool" as opposed to "Install the AWS CLI tool and git" when you look at the the alphabetized package names immediately below it.
```suggestion
    - name: Install the AWS CLI tool and git
```